model/src/application/component/mod.rs

```diff
@@ -555,6 +560,11 @@ impl Serialize for Component {
                     state.serialize_field("value", &text_input.value)?;
                 }
             }
+            // We are not serializing all fields so this will fail to deserialize. But it is all that can be done
+            // to avoid losing incoming messages at this time
```

Comment:
can we wrap this to 80 characters:
```suggestion
            // We are not serializing all fields so this will fail to
            // deserialize. But it is all that can be done to avoid losing
            // incoming messages at this time.
```